pymc3/stats.py

```diff
@@ -58,21 +58,35 @@ def wrapped_f(pymc3_obj, *args, **kwargs):
 @statfunc
 def autocorr(x, lag=1):
     """Sample autocorrelation at specified lag.
-    The autocorrelation is the correlation of x_i with x_{i+lag}.
-    """
 
+    Parameters
+    ----------
+    x : Numpy array
+        An array containing MCMC samples
+    lag : int
+        The desidered lag to take in consideration
+    """
     S = autocov(x, lag)
     return S[0, 1] / np.sqrt(np.prod(np.diag(S)))
 
 
 @statfunc
 def autocov(x, lag=1):
-    """
-    Sample autocovariance at specified lag.
-    The autocovariance is a 2x2 matrix with the variances of
-    x[:-lag] and x[lag:] in the diagonal and the autocovariance
-    on the off-diagonal.
-    """
+    """Sample autocovariance at specified lag.
```

Comment:
As far as I see documentation should be """Comment
ref: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html